common/src/main/java/com/wynntils/handlers/chat/ChatHandler.java

```diff
@@ -150,7 +150,7 @@ public void onSystemChatReceived(ChatPacketReceivedEvent.System event) {
     public void onStatusEffectUpdate(MobEffectEvent.Update event) {
         if (event.getEntity() != McUtils.player()) return;
 
-        if (event.getEffect() == MobEffects.MOVEMENT_SLOWDOWN
+        if (event.getEffect() == MobEffects.MOVEMENT_SLOWDOWN.value()
```

Comment:
Is this a bug fix? I can't see how both the code before and after could be valid, so it's not a refactoring.

Reply:
I juuuust noticed and was about to reply to myself. :)